libraries/pipes/implementations/rust/flake.nix

```diff
@@ -0,0 +1,41 @@
+{
+  description = "Dagster Pipes Rust flake";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    rust-overlay.url = "github:oxalica/rust-overlay";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, rust-overlay, flake-utils, ... }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        overlays = [ (import rust-overlay) ];
+        pkgs = import nixpkgs {
+          inherit system overlays;
+        };
+      in
+      with pkgs;
+      {
+        devShells.default = mkShell {
+          buildInputs = [
+            git
+            openssl
+            uv
+            pkg-config
+            (
+              rust-bin.selectLatestNightlyWith (toolchain: toolchain.default.override {
+                extensions = [
+                  "rust-src"
+                  "rust-analyzer"
+                ];
+              })
+            )
+          ];
+          shellHook = ''
```

Comment:
I found [these](https://github.com/dagster-io/community-integrations/blob/main/libraries/pipes/implementations/java/dagster-pipes-java/nix/shells/default/default.nix#L17) exports to be useful when using Python coming from Nix.